### DISCUSSION TITLE
Code name for the next major Debian release after bookworm is trixie.

### DISCUSSION THREAD

**Discussion by tteck** [tteck/Proxmox#2155] [Announcements]
To **_test_** out trixie, create a new Debian 12 LXC, then execute the following in the console
```
sed -i 's/bookworm/trixie/g' /etc/apt/sources.list
apt-get update && DEBIAN_FRONTEND=noninteractive apt-get -o Dpkg::Options::="--force-confold" dist-upgrade -y
```
reboot

Debian GNU/Linux trixie/sid debian tty1

**Comment by oOStroudyOo** [comment]
I might try this on a fully running container with important apps on because I'm a mad man.
Go big or go home I guess.

I'll let you know if Proxmox dies.

EDIT:
Seems to install fine

**Reply by tteck** [reply]
all the new versions
